homeassistant/components/systemmonitor/sensor.py

```diff
@@ -60,62 +66,188 @@
+        name="Disk use (percent)",
+        native_unit_of_measurement=PERCENTAGE,
+        icon="mdi:harddisk",
+        state_class=STATE_CLASS_TOTAL,
+    ),
+    "ipv4_address": SysMonitorSensorEntityDescription(
+        key="ipv4_address",
+        name="IPv4 address",
+        native_unit_of_measurement="",
+        icon="mdi:server-network",
+        mandatory_arg=True,
+    ),
+    "ipv6_address": SysMonitorSensorEntityDescription(
+        key="ipv6_address",
+        name="IPv6 address",
+        native_unit_of_measurement="",
+        icon="mdi:server-network",
+        mandatory_arg=True,
+    ),
+    "last_boot": SysMonitorSensorEntityDescription(
+        key="last_boot",
+        name="Last boot",
+        device_class=DEVICE_CLASS_TIMESTAMP,
+    ),
+    "load_15m": SysMonitorSensorEntityDescription(
+        key="load_15m",
+        name="Load (15m)",
+        native_unit_of_measurement=" ",
+        icon=CPU_ICON,
+        state_class=STATE_CLASS_TOTAL,
+    ),
+    "load_1m": SysMonitorSensorEntityDescription(
+        key="load_1m",
+        name="Load (1m)",
+        native_unit_of_measurement=" ",
+        icon=CPU_ICON,
+        state_class=STATE_CLASS_TOTAL,
+    ),
+    "load_5m": SysMonitorSensorEntityDescription(
+        key="load_5m",
+        name="Load (5m)",
+        native_unit_of_measurement=" ",
+        icon=CPU_ICON,
+        state_class=STATE_CLASS_TOTAL,
+    ),
+    "memory_free": SysMonitorSensorEntityDescription(
+        key="memory_free",
+        name="Memory free",
+        native_unit_of_measurement=DATA_MEBIBYTES,
+        icon="mdi:memory",
+        state_class=STATE_CLASS_TOTAL,
+    ),
+    "memory_use": SysMonitorSensorEntityDescription(
+        key="memory_use",
+        name="Memory use",
+        native_unit_of_measurement=DATA_MEBIBYTES,
+        icon="mdi:memory",
+        state_class=STATE_CLASS_TOTAL,
+    ),
+    "memory_use_percent": SysMonitorSensorEntityDescription(
+        key="memory_use_percent",
+        name="Memory use (percent)",
+        native_unit_of_measurement=PERCENTAGE,
+        icon="mdi:memory",
+        state_class=STATE_CLASS_TOTAL,
+    ),
+    "network_in": SysMonitorSensorEntityDescription(
+        key="network_in",
+        name="Network in",
+        native_unit_of_measurement=DATA_MEBIBYTES,
+        icon="mdi:server-network",
+        state_class=STATE_CLASS_TOTAL_INCREASING,
+        mandatory_arg=True,
+    ),
+    "network_out": SysMonitorSensorEntityDescription(
+        key="network_out",
+        name="Network out",
+        native_unit_of_measurement=DATA_MEBIBYTES,
+        icon="mdi:server-network",
+        state_class=STATE_CLASS_TOTAL_INCREASING,
+        mandatory_arg=True,
+    ),
+    "packets_in": SysMonitorSensorEntityDescription(
+        key="packets_in",
+        name="Packets in",
+        native_unit_of_measurement=" ",
+        icon="mdi:server-network",
+        state_class=STATE_CLASS_TOTAL_INCREASING,
+        mandatory_arg=True,
```

Comment:
We should really disable some by default that are less common to be useful, like the packets one (since we have the bytes on already)

Reply:
there is no sensor enabled by default, since the user needs to explicit enable them by set in configuration (see [docs](https://www.home-assistant.io/integrations/systemmonitor/))